packages/editor/src/components/block-styles/index.js

```diff
@@ -72,7 +71,7 @@ function BlockStyles( {
 	onSwitch = noop,
 	onHoverClassName = noop,
 } ) {
-	if ( ! styles ) {
+	if ( ! styles || styles.length === 0 ) {
```

Comment:
Even if the block has no styles at all, it contains an empty array of styles. Should we continue to check for the length?

Reply:
yes, the idea is that we don't show this panel at all if there's no styles.